src/libgpu/latte/latte_enum_sq.h

```diff
@@ -247,6 +247,7 @@ ENUM_BEG(SQ_SEL, uint32_t)
    ENUM_VALUE(SEL_W,                      3)
    ENUM_VALUE(SEL_0,                      4)
    ENUM_VALUE(SEL_1,                      5)
+   ENUM_VALUE(SEL_BIAS,                      6)
```

Comment:
Should I fix these style problems and then recommit? I am quite new at GitHub stuffs, so many thanks for any advise and guidance.
And Thanks for all your detailed advice.